File: src/main/java/hdzi/editstarters/initializr/InitializrBom.java
package hdzi.editstarters.initializr;

import hdzi.editstarters.dependency.IBom;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class InitializrBom implements IBom {
    private String groupId;
    private String artifactId;
    private String version;
    private List<String> repositories;
}

File: src/main/kotlin/hdzi/editstarters/springboot/initializr/InitializrBom.kt
package hdzi.editstarters.springboot.initializr

import hdzi.editstarters.springboot.Bom

class InitializrBom : Bom {
    override lateinit var groupId: String
    override lateinit var artifactId: String
    lateinit var version: String
    lateinit var repositories: List<String>
}